app/helpers/patient_details_helper.rb

```diff
@@ -12,6 +12,7 @@ module PatientDetailsHelper # rubocop:todo Metrics/ModuleLength
   AFTERNOON_CONTACT_WINDOW = (12..16).freeze
   EVENING_CONTACT_WINDOW = (16..19).freeze
   UNSPECIFIED_CONTACT_WINDOW = (12..16).freeze
+  CUSTOM_CONTACT_OPTIONS = (0..23).to_a.map(&:to_s).freeze
```

Comment:
I think if you're feeling dangerous, just `('0'..'23').to_a` should work.

Reply:
TIL

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1019/commits/daa0195b5441006dc8259d33d13da45f325d66c6#diff-6a782ad659c7d687b8b004c762706d9c4c1ee24674305f8df71babe430abdb73R15